source/UIABrowseMode.py

```diff
@@ -99,13 +99,7 @@ def UIAControlQuicknavIterator(itemType,document,position,UIACondition,direction
 		walker=UIAHandler.handler.clientObject.createTreeWalker(UIACondition)
 		# some implementations (Edge, Word) do not correctly  class embedded objects (graphics, checkboxes) as being the enclosing element, even when the range is completely within them. Rather, they still list the object in getChildren.
 		# Thus we must check getChildren before getEnclosingElement.

```

Comment:
This comment doesn't seem useful/correct now that you have the UIAElementAtStart property. There's no change to what happens in the end, but the decision is now implementation specific. Should the comment be removed?